build/publish.sh

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash
+
+# !!! 
+# Note: This script is currently not in use. Multi-platform builds cannot be stored locally, but only
+# in a Docker registry, so we currently add the appropriate tags and push to the registry at build
+# time in build.sh. 
+# !!!!
```

Comment:
If it is not used, can we remove the file?

Also: I still have difficulties bringing multi platform builds into this context. It makes the change very complex.

Reply:
Yes, will do. Kept this in for reference in this review mostly.